document/api/src/main/java/io/camunda/document/api/DocumentStoreClient.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.document.api;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.function.Function;
+
+public interface DocumentStoreClient {
+
+  String resolveStoreId(String storeId);
```

Comment:
Curious what the purpose of this is? Do we have different ids per store?

Reply:
The request may use a non-specific store ID such as `null` or `"default"` that is technically resolvable by the store wrapper, but the document reference should contain a specific store ID.

I'll try to provide the resolved ID as part of the return value instead, this should look nicer, wdyt?